web/vtadmin/src/index.css

```diff
@@ -14,8 +14,6 @@
  * limitations under the License.
  */
 
-@import './style/fonts.css';
```

Comment:
This change is just for @planetscale/vtadmin and is important to making asset resolution in our Rails app easier. This likely won't run into any merge conflicts with OS VTAdmin.